Signal/src/UserInterface/Strings.swift

```diff
@@ -1,12 +1,14 @@
-//  Created by Michael Kirk on 12/29/16.
-//  Copyright © 2016 Open Whisper Systems. All rights reserved.
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
 
 import Foundation
 
-@objc class Strings: NSObject {
-    @objc class Calls: NSObject {
-        static let callBackButtonTitle = NSLocalizedString("CALLBACK_BUTTON_TITLE", comment: "notification action")
-        static let missedCallNotificationBody = NSLocalizedString("MISSED_CALL", comment: "notification title")
-        static let missedCallNotificationBodyWithCallerName = NSLocalizedString("MSGVIEW_MISSED_CALL", comment: "notification title. Embeds {{Caller's Name}}")
-    }
+/**
+ * Strings re-used in multiple places should be added here.
+ */
+@objc class CallStrings: NSObject {
+    static let callBackButtonTitle = NSLocalizedString("CALLBACK_BUTTON_TITLE", comment: "notification action")
+    static let missedCallNotificationBody = NSLocalizedString("MISSED_CALL", comment: "notification title")
+    static let missedCallNotificationBodyWithCallerName = NSLocalizedString("MSGVIEW_MISSED_CALL", comment: "notification title. Embeds {{Caller's Name}}")
```

Comment:
I tend to agree, but I think in this case it makes sense to re-use, since the distinction is between "call back" a redphone call and "call back" an RTC call.